src/wp-includes/block-supports/duotone.php

```diff
@@ -313,6 +313,92 @@ function wp_register_duotone_support( $block_type ) {
 		}
 	}
 }
+/**
+ * Renders the duotone filter SVG and returns the CSS filter property to
+ * reference the rendered SVG.
+ *
+ * @param array $preset Duotone preset value as seen in theme.json.
+ *
+ * @return string Duotone CSS filter property.
+ */
```

Comment:
```suggestion
/**
 * Renders the duotone filter SVG and returns the CSS filter property to
 * reference the rendered SVG.
 *
 * @since 5.9.0
 *
 * @param array $preset Duotone preset value as seen in theme.json.
 * @return string Duotone CSS filter property.
 */
```